doc/nrf/ug_nrf52.rst

```diff
@@ -122,6 +122,20 @@ In addition, you can run the :ref:`zephyr:bluetooth-samples` that are included f
 
 For available libraries, see :ref:`lib_bluetooth_services` (|NCS|) and :ref:`zephyr:bluetooth_api` (Zephyr).
 
+Bluetooth mesh
+==============
+
+Bluetooth mesh is a low-power mesh networking solution based on Bluetooth Low Energy, optimized for creating large-scale device networks.
+It is developed by the Bluetooth® Special Interest Group (SIG), and is compatible with core specification version 4.0 and higher.
```

Comment:
We say that we support core specification for BLE, but is the interesting. It would be more interesting to see that we support Bluetooth mesh core/model spec 1.0.1.

Reply:
Same for first sentence for the 53.